reactor-core/src/blockHoundTest/java/reactor/core/scheduler/ReactorBlockHoundIntegrationTest.java

```diff
@@ -74,26 +74,50 @@ public void shouldDetectBlockingCallsInOperators() {
 	}
 
 	@Test
-	public void shouldNotReportScheduledFutureTask() {
-		for (int i = 0; i < 1_000; i++) {
+	public void shouldNotReportSchedulerScheduledFutureTask() {
+		for (int i = 0; i < 10_000; i++) {
 			Scheduler taskScheduler = Schedulers.newSingle("foo");
 			try {
 				Runnable dummyRunnable = () -> {
 				};
 
 				for (int j = 0; j < 257; j++) {
-					taskScheduler.schedule(dummyRunnable, 200, TimeUnit.MILLISECONDS);
+					taskScheduler.schedule(dummyRunnable, j + 1, TimeUnit.MILLISECONDS);
 				}
 				Disposable disposable = taskScheduler.schedule(dummyRunnable, 1, TimeUnit.SECONDS);
 
-				RaceTestUtils.race(disposable::dispose, disposable::dispose);
+				RaceTestUtils.race(Schedulers.parallel(), disposable::dispose);
 			}
 			finally {
 				taskScheduler.dispose();
 			}
 		}
 	}
 
+	@Test
+	public void shouldNotReportWorkerScheduledFutureTask() {
+		for (int i = 0; i < 10_000; i++) {
```

Comment:
On my local machine `10_000` was reliable in producing the exception -- `1_000` not so much. Let me know if this is acceptable.

Reply:
As a follow up to this comment, I created https://github.com/reactor/reactor-core/issues/3216 which would create an opportunity to use a more heavy crunching when run locally and do a quicker run-through via CI.